### ISSUE TITLE
Use more SSR

### ISSUE THREAD

**Issue by aalemayhu** [2anki/server#112]
We can increase the performance (initial load time) of the site overall by using more server side rendered pages.

**Comment by aalemayhu** [comment]
You can read about SSR here (we are not using NuxtJS) https://nuxtjs.org/guides/concepts/server-side-rendering/

**Comment by aalemayhu** [comment]
Reopening after triage: closure was a stale-bot auto-close (not a product decision). Reviewed by: orchestrator (stale-bot override rule).

**Comment by aalemayhu** [comment]
The landing pages are prerendered to static HTML at build time now, so the initial load is served as ready-made markup rather than waiting on the client to render. That covers the load-time goal here, so I'm closing this. If there's a specific page that still feels slow to first paint, open a new issue naming it and I'll look.
